### DISCUSSION TITLE
Is it possible to label an area of a mesh or paint on a mesh?

### DISCUSSION THREAD

**Discussion by FluencyLiu** [pyvista/pyvista#1884] [Q&A]
## Description

I'm trying labeling the tooth area on oral scan meshes. I uploaded an example of the oral scan, and a tooth is labeled with red on this example.
I'm using the z-painting function of Meshlab to do it, but it is not that handy.
Is it possible to label an area of a mesh or paint on a mesh with pyvista? Thx.

## Example Data

![example](https://user-images.githubusercontent.com/8326264/140511461-88a9b0e9-6247-49a2-b5c1-406a7f99c4df.jpg)
This is an oral scan example in stl format.
[example.zip](https://github.com/pyvista/pyvista-support/files/7486041/example.zip)

**Comment by banesullivan** [comment]
Here's an example of how to pick a region of a mesh.

```py
import pyvista as pv

mesh = pv.read('example.stl')

p = pv.Plotter(notebook=0)
p.add_mesh(mesh)
p.enable_cell_picking(through=False)
p.show()
```

https://user-images.githubusercontent.com/22067021/143785581-c4c43107-d18b-4a46-807c-db9e3d1fe913.mov
